### PR TITLE
Bugfix MTE-577 [v111] Temporary fix for BookmarkingTests

### PR DESCRIPTION
**Pull request by clarmso**
`testBookmarksAwesomeBar()` from `BookmarkingTests` has been failing intermittently at least on Bitrise. We have observed that `www.ebay.com` failed to load completely within the given timeout.

The use of external assets is still to be discussed and be addressed within the mobile test engineering team. Let me use some websites that we "know" 😛 better and may have better availability for now.

### CODE REVIEW

File: Tests/XCUITests/BookmarkingTests.swift [hunk 1]
```diff
@@ -145,7 +145,7 @@ class BookmarkingTests: BaseTestCase {
         waitForTabsButton()
         navigator.performAction(Action.OpenNewTabFromTabTray)
         navigator.goto(URLBarOpen)
-        typeOnSearchBar(text: "http://www.olx.ro")
+        typeOnSearchBar(text: "https://mozilla.org")
```
Comment (clarmso) [inline Tests/XCUITests/BookmarkingTests.swift:148]: Mozilla isn't among the bookmarked sites, shortcuts or pinned sites. 🦖

File: Tests/XCUITests/BookmarkingTests.swift [hunk 2]
```diff
@@ -133,10 +133,10 @@ class BookmarkingTests: BaseTestCase {
         XCTExpectFailure("The app was not launched", strict: false) {
             waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
         }
-        typeOnSearchBar(text: "www.ebay")
+        typeOnSearchBar(text: "www.google")
         waitForExistence(app.tables["SiteTable"])
-        waitForExistence(app.tables["SiteTable"].cells.staticTexts["www.ebay"], timeout: 5)
-        XCTAssertTrue(app.tables["SiteTable"].cells.staticTexts["www.ebay"].exists)
+        waitForExistence(app.tables["SiteTable"].cells.staticTexts["www.google"], timeout: 5)
```
Comment (clarmso) [inline Tests/XCUITests/BookmarkingTests.swift:138]: I bet Google has better uptime than EBay. If Google is down, we have no work done anyway.

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Edited 1 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 0 files</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 937ab43a8c6aa96aaeb0cc2a2db85acdbfb57573
</p>

**Comment by clarmso** [comment]
The failed tests indicated in the status above passed on retry. None of the failed tests is the updated test.
https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/27daa931-935d-450a-a350-7d7fdf3485cf

**Comment by clarmso** [comment]
@isabelrios Could you please kindly land this PR? Bitrise was passing before I submitted the empty commit.
https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/27daa931-935d-450a-a350-7d7fdf3485cf

**Review by isabelrios — APPROVED** [review]

**Comment by isabelrios** [comment]
> @isabelrios Could you please kindly land this PR? Bitrise was passing before I submitted the empty commit. https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/27daa931-935d-450a-a350-7d7fdf3485cf

I can't land it without running it on Bitrise... I started the build, we will have another try to see that the tests pass.

**Pull request merged**